src/metalkit.cs

```diff
@@ -298,6 +321,20 @@ interface MTKTextureLoader {
 		[Async]
 		void FromNames (string[] names, nfloat scaleFactor, [NullAllowed] NSBundle bundle, [NullAllowed] MTKTextureLoaderOptions options, MTKTextureLoaderArrayCallback completionHandler);
 
+		[NoiOS]
+		[NoTV]
+		[Mac (10,12, onlyOn64 : true)]
+		[Async]
+		[Export ("newTexturesWithNames:scaleFactor:displayGamut:bundle:options:completionHandler:")]
+		void FromNames (string[] names, nfloat scaleFactor, NSDisplayGamut displayGamut, [NullAllowed] NSBundle bundle, [NullAllowed] NSDictionary options, MTKTextureLoaderArrayCallback completionHandler);
+
+		[NoiOS]
+		[NoTV]
+		[Mac (10,12, onlyOn64 : true)]
+		[Wrap ("FromNames (names, scaleFactor, displayGamut, bundle, options == null ? null : options.Dictionary, completionHandler)")]
```

Comment:
Really minor (and no need to change the PR): `options?.Dictionary` is shorter than `options == null ? null : options.Dictionary` 😄

Reply:
And I just saw @dalexsoto made the exact same comment on the original PR 😁